### PR TITLE
Create dockerfile and dockercompose

### PR DESCRIPTION
**Pull request by KacperST**

### CODE REVIEW

File: backend/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      - postgres
+
+
+  postgres:
+    image: postgres:16-bullseye
+    env_file:
+      - .env
+    ports:
+      - 5432:5432
+
```
Comment (michalszc) [inline backend/docker-compose.yml]: missing newline

File: backend/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+fastapi==0.110.1
+psycopg2-binary==2.9.9
+pydantic==2.7.0
+SQLAlchemy==2.0.29
+uvicorn==0.29.0
```
Comment (michalszc) [inline backend/requirements.txt]: missing newline

**Review by michalszc — CHANGES_REQUESTED** [review]

File: backend/requirements.txt (hunk 2 above)
Comment (KacperST) [reply backend/requirements.txt]: Added

**Review by Glonk223 — APPROVED** [review]
Looks good

**Review by michalszc — APPROVED** [review]

**Pull request merged**